cli/src/subcommand/init.rs

```diff
@@ -15,13 +15,19 @@ struct InitPromptResult {
 
 fn init_prompt() -> Result<InitPromptResult, Error> {
     // TODO: Add support for finding root of the project
-    let is_config_file_exists = std::path::Path::new(DEFAULT_CONFIG_FILE_PATH).exists();
+    let existing_configs: Vec<&str> = DEFAULT_CONFIG_FILE_PATHS
```

Comment:
imo. If the `ConfigBuilder` can check for the existence of the config file, we might not have to worry about variations in the YAML file extension when implementing the init command. 💭

Reply:
Agreed on the shared logic. changed to move it to ConfigBuilder by e1d3198f6ced （is this what you were suggesting?